src/plugins/analysis/architecture_detection/view/cpu_architecture.html

```diff
@@ -2,21 +2,23 @@
 
 {% block analysis_result_details %}
 
-<tr>
-	<td class="table-head-light"><b>Legend</b></td>
-	<td>(M): Detected by meta data; (C): Detected by codescanner (Binary analysis plug-in)</td>
-</tr>
 <tr>
 	<td class="table-head-light"><b>Architecture</b></td>
 	<td class="table-head-light"><b>Means of Detection</b></td>
 </tr>
-	{% for key in firmware.processed_analysis[selected_analysis] %}
-    	{% if key | is_not_mandatory_analysis_entry %}
-			<tr>
-				<td> {{ key }} </td>
-				<td>{{ firmware.processed_analysis[selected_analysis][key] }}</td>
-			</tr>
-		{% endif %}
-	{% endfor %}
+	{% if "architectures" in firmware.processed_analysis[selected_analysis] %}
+	 	{% for result, method in firmware.processed_analysis[selected_analysis]["architectures"].items() %}
+	 		<tr>
+	 			<td>{{ result }}</td>
+	 			<td>{{ method }}</td>
+	 		</tr>
+		{% endfor %}
+	{% else %}
+		<tr>
+			<td></td>
+			<td>The analysis result is outdated, please rerun the analysis.</td>
```

Comment:
I'm not happy with checking in the html if the analysis result is outdated.
What is the correct way to make breaking changes in plugins?